utils/misc.mli

```diff
@@ -257,6 +257,14 @@ val protect_writing_to_file
           channel. If the function raises an exception then [filename]
           will be removed. *)
 
+(** [concat_null_terminated [x1;x2; ... xn]] is
+    [x1 ^ "\000" ^ x2 ^ "\000" ^ ... ^ xn ^ "\000"] *)
+val concat_null_terminated : string list -> string
```

Comment:
The rest of the comments in this file are placed _after_ the corresponding value.

Reply:
done